src/sass/components/govcy-alerts.scss

```diff
@@ -68,10 +68,10 @@
     @extend .alert;
```

Comment:
I think there is a small problem with the `govcy-all-alerts` mixin. It looks like it's rendering a small white line at the edge. See screenshot

Reply:
<img width="585" alt="Screenshot 2021-12-13 at 10 10 25" src="https://user-images.githubusercontent.com/3469840/145793269-35c9c740-a2b3-450a-a059-15ab8b93d98c.png">